app/config.py

```diff
@@ -36,6 +36,7 @@ class Config(object):
     NR_BROWSER_KEY = getenv("NR_BROWSER_KEY")
     settings = newrelic.agent.global_settings()
     NR_MONITOR_ON = settings and settings.monitor_mode
+    COMMIT_HASH = getenv("COMMIT_HASH", "Unknown")
```

Comment:
Let's change `Unknown` to several dashes instead:

```suggestion
    COMMIT_HASH = getenv("COMMIT_HASH", "-----")
```